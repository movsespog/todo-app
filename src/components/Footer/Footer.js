import React from 'react';
import PropTypes from 'prop-types';

import './Footer.css';

const footerPropTypes = {
  className: PropTypes.string.isRequired,
};

const Footer = ({ className }) => (
  <div className={className}>
    <div className="footer__container">

    </div>
  </div>
);

Footer.propTypes = footerPropTypes;

export default Footer;
