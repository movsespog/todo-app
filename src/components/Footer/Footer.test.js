import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

test('Footer gets rendered with a given classname', () => {
  const className = 'test-classname';
  const component = renderer.create(<Footer className={className} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree.props.className).toEqual(className);
});
