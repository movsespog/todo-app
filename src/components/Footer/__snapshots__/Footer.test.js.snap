// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer gets rendered with a given classname 1`] = `
<div
  className="test-classname"
>
  <div
    className="footer__container"
  >
    <p
      className="footer__p"
    >
      Made using React + Redux -
       
      <a
        href="https://github.com/arthur-melo"
        rel="noopener noreferrer"
        target="_blank"
      >
        @arthur-melo
      </a>
    </p>
  </div>
</div>
`;
